CMakeLists.txt

```diff
@@ -957,6 +968,7 @@ if(${PLAYER_BUILD_EXECUTABLE} AND ${PLAYER_TARGET_PLATFORM} STREQUAL "SDL2")
 		string(REPLACE "." "," RC_VERSION ${PROJECT_VERSION})
 		string(TIMESTAMP RC_YEAR "%Y")
 		configure_file(resources/windows/player.rc.in resources/windows/player.rc @ONLY)
+		file(COPY resources/windows DESTINATION resources PATTERN "*.in" EXCLUDE)
```

Comment:
will try to figure out if this is solvable without the copy

Reply:
According to the docs (which is peak MSDN again translating names of Resource fields xD) the resources can also be absolute paths.

So this should work:
```
23456 ICON "@CMAKE_CURRENT_SOURCE_DIR@/resources/windows/player.ico"
```

Same for the XML line (which is currently MinGW only but will need the manifest for HighDPI later):

```
1 24 "@CMAKE_CURRENT_SOURCE_DIR@/resources/windows/player.xml"
```